runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```diff
@@ -263,6 +263,12 @@
       "icon": 1660,
       "name": "Ruby Necklace",
       "xp": 75
+    },
+	{
```

Comment:
Format this consistently with the rest of the file.

Reply:
@Adam- same as #4334